### PR TITLE
fix: sidebar bottom section design change

### LINKED ISSUE #24855: [Task][Dev]: Too much space below the bottom sticky section on left panel
### Is there an existing issue for this?

- [X] I have searched the existing issues

### SubTasks

Designs https://github.com/appsmithorg/appsmith/issues/24099 & [notion](https://www.notion.so/appsmith/Task-Too-much-space-below-the-bottom-sticky-section-on-left-panel-71301f295bc1477a92ddf6e0489a543e)

### PR DESCRIPTION
**Pull request by albinAppsmith**
## Description

This PR updates the design of bottom section in sidebar where the version of appsmith and release time.

#### PR fixes following issue(s)
Fixes #24855

#### Type of change

- New feature (non-breaking change which adds functionality)

- [ ] Manual
- [ ] Jest
- [ ] Cypress
>
>
#### Dev activity
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] PR is being merged under a feature flag

#### QA activity:
- [ ] [Speedbreak features](https://github.com/appsmithorg/TestSmith/wiki/Guidelines-for-test-plans#speedbreakers-) have been covered
- [ ] Test plan covers all impacted features and [areas of interest](https://github.com/appsmithorg/TestSmith/wiki/Guidelines-for-test-plans#areas-of-interest-)
- [ ] Test plan has been peer reviewed by project stakeholders and other QA members
- [ ] Manually tested functionality on DP
- [ ] We had an implementation alignment call with stakeholders post QA Round 2
- [ ] Cypress test cases have been added and approved by SDET/manual QA
- [ ] Added `Test Plan Approved` label after Cypress tests were reviewed
- [ ] Added `Test Plan Approved` label after JUnit tests were reviewed

### CODE REVIEW

**Review by akash-codemonk — APPROVED** [review]

**Review by akash-codemonk — APPROVED** [review]

**Comment by shadabbuchh** [comment]
Issues in the PR:

1. Font size has to be 12 px as per Notion & figma.
2. Font colour has to be ##6A7585

File: app/client/src/ce/pages/Home/LeftPaneBottomSection.tsx [hunk 1]
```diff
@@ -124,7 +133,7 @@ function LeftPaneBottomSection() {
               cloudHosting,
             )}
           </span>
-          {howMuchTimeBefore !== "" && (
+          {howMuchTimeBefore === "" && (
```
Comment (akash-codemonk) [inline app/client/src/ce/pages/Home/LeftPaneBottomSection.tsx:136]: Undo?

Comment (albinAppsmith) [reply app/client/src/ce/pages/Home/LeftPaneBottomSection.tsx]: Yes, sorry I pushed it by mistake.

**Review by akash-codemonk — DISMISSED** [review]

**Comment by shadabbuchh** [comment]
if we go with font size as 12px, there is a line break that does not look right.

**Comment by Sripriya93** [comment]
We have few discrepancies with the padding, width and height, Apart from that Font size, color and background color are fine.
QA completed. Moving to done.

**Review by akash-codemonk — APPROVED** [review]

**Pull request merged**